packages/vscode-extension/package.json

```diff
@@ -246,6 +268,21 @@
           "command": "fx-extension.openHelpFeedbackLink",
           "when": "view == teamsfx-help-and-feedback",
           "group": "navigation@1"
+        },
+        {
+          "command": "fx-extension.openOfficeDevDevelopmentLink",
```

Comment:
Usually, this kind of command is disabled in command palette (contributes.menus.commandPalette)